runtimes/java-11.0/src/main/java/io/openruntimes/java/Server.java

```diff
@@ -44,7 +44,7 @@ public static Resp execute(Req req, Resp resp) {
             Map<String, Object> output = new HashMap<String, Object>();
             output.put("response", userResponse.data);
             output.put("stdout", outStream.toString());
-            return resp.result(RuntimeResponse.gson.toJson(output));
+            return resp.code(userResponse.statusCode).result(RuntimeResponse.gson.toJson(output));
```

Comment:
@lohanidamodar or @abnegate, would either of you confirm this is something we should add?

Reply:
Yep we should. From a quick glance, we'll need to fix this for other runtimes too (only saw it applied in node, php and python)

Reply:
Is that something that I should do in this PR as well? I would be down, I just wanted to make sure that I don't blow things out of scope :)

Reply:
Should be separate PR's to keep the scope in check as you mentioned, but if you're willing to make the changes for some of the other runtimes that would be awesome :grin:

Reply:
I will see what I can do around the weekend! I reverted the unauthorized code change from above; if anything else is missing, let me know!

Reply:
@geisterfurz007, we typically make a change for all languages at the same time to make sure they're all in sync.

Reply:
@stnguyen90 Ok, just to clarify: Should I change the other runtimes in this one as well now or not. Because @abnegate's comment indicated the contrary 😅

Reply:
@geisterfurz007 let's keep this PR's scope small.

Reply:
Works for me! Checking through the thread I think there is nothing left for me to do then anymore; the unauthorized code is set to 500 and the changes to other runtimes are to be done in a separate PR (not sure yet if / when I will find the time for that, sadly).

Let me know if there is anything you need from me!